WAGTAIL3UPGRADE.md

```diff
@@ -0,0 +1,30 @@
+# Wagtail 3.0 upgrade issues
+
+## Admin sidebar
+
+### Current Implementation
+
+It is customised to hide the search bar by user group/permissions (Students)
+
+### Problem
+
+The side bar is now a javascript constructed element so the current implementation doesn't work.
+
+### File:
+
+`rca/account_management/templates/wagtailadmin/base.html`
+
+
+## Content Panels
+
+### Current Implementation
+
+`StudentPage` which uses a `PerUserPageMixin` to hide/show content panels based on group/user permissions
+
+### Problem
+
+I'm unsure about the changes needed here. I'm not entirely sure how this is intended or currently works.
+
+### Files:
+
+`rca/people/models.py` & `rca/people/utils.py`
```

Comment:
Thanks Kevin. This sounds like it's possible within the new Wagtail version, I will investigate it.

Reply:
Did you get anywhere with this @nickmoreton ?

Reply:
@kevinhowbrook I've not done any further work on it to date.

Reply:
For the field permissions stuff I have added some notes to this support ticket on how this can be resolved/reworked regarding the updates https://projects.torchbox.com/projects/hosting-application-support/tickets/30#update-70425155

Reply:
Thanks @kevinhowbrook

Reply:
I've done a bunch of changes now that hopefully is a solution. See 25th Oct commits